docs/api/version-history.md

```diff
@@ -15,7 +15,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 ## v1.48 API changes
 
-[Docker Engine API v1.48](https://docs.docker.com/engine/api/v1.48/) documentation
+[Docker Engine API v1.48](https://docs.docker.com/reference/api/engine/version/v1.48/) documentation
```

Comment:
This one is returning 404. Is that just a gap on the documentation side?

Reply:
Oh! Yes that's expected to be a 404 for now. This is the upcoming API version for v28.0.0, so the docs doesn't have that one yet until we release v28.0.0. 😅